packages/validation/src/account.schema.ts

```diff
@@ -4,7 +4,9 @@ export const updateAccountSchema = object({
   email: string().max(128, 'Email must be 128 characters or less.'),
   address_1: string().max(64, 'Address must be 64 characters or less.'),
   city: string().max(24, 'City must be 24 characters or less.'),
-  company: string().max(128, 'Company must be 128 characters or less.'),
+  company: string()
+    .max(128, 'Company must be 128 characters or less.')
+    .nullable(true),
```

Comment:
I think nullable defaults to true if it is used.

```suggestion
    .nullable(),
```